scc/Cargo.toml

```diff
@@ -15,6 +15,8 @@ toml = "0.5"
 vmap = { version = "0.5", default-features = false, optional = true }
 fd-lock = "3"
 msgbox = { version = "0.7", optional = true }
+bpaf = "0.7.10"
```

Comment:
we can set the version to `0.7`, we don't need to specify the patch since they should be API compatible